README.md

```diff
@@ -94,7 +94,6 @@ Where available commands are:
 | remove       | remove package(s) |
 | search       | look for package(s) |
 | sysinfo      | show system information in markdown style |
-| test         | test crew command(s) |
```

Comment:
What if we want to test prior to submitting a PR?  If this is removed, we would have to submit a PR and see if the tests fail first?  If that is true, not ideal.

Reply:
If we want to test prior to submitting a PR, we now run `rake test`.

Reply:
This also allows easier testing on non-ChromeOS environments, as you do not need to have crew installed to test every command at once.